controllers/usersController.js

```diff
@@ -110,6 +110,37 @@ class UsersController {
     return user;
   }
 
+  /**
+ * Updates the password of a user.
+ *
+ * @static
+ * @param {object} request The express request object
+ * @param {object} response The express response object
+ * @param {function} next The express next function
+ * @returns {void}
+ * @memberof UsersController
+ */
+  static async resetPassword(request, response, next) {
+    try {
+      const count = await User.update(
+        {
+          password: request.body.password
+        },
+        {
+          where: { email: request.recoveryEmail }
```

Comment:
#### Nitpick
- `request.recoveryEmail`?

Reply:
I have refactored this to include the recovery email in the body object instead.